src/main/java/com/toptrumps/core/Card.java

```diff
@@ -1,73 +1,25 @@
 package com.toptrumps.core;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-
-// TODO: Refactor this class
 public class Card {
     // instance variables
     private String description;
-    private HashMap<String, Integer> cardValuesByCat;
-    private ArrayList<String> bestCat;
-
-
-    public Card(String description) {
-
+    private Attribute[] attributes;
```

Comment:
@rjs68 maybe it's a good idea to use ArrayList instead of Arrays! It will make the comparison logic simpler later on.

Reply:
Yes I agree with this - hadn't considered the future impact! Made the change now